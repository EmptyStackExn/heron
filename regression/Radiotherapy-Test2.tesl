// Radiotherapy machine

Q-clock hr
Q-clock gy
Q-clock start sporadic 0.0
Q-clock stop
Q-clock emstop

tag relation gy = 1.5 * hr
tag relation gy = start
tag relation start = stop
tag relation stop = emstop

emstop implies stop

start time delayed by 2.0 on gy implies stop

emstop sporadic 0.5 on hr

@policy asap
@run
@print
@output tikz select hr gy start stop emstop
