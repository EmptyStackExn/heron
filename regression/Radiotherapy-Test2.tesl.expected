  \node       at (1+0.1, -4+0.1)  {\Cross} ;
  \node       at (2+0.1, -2+0.1)  {\Cross} ;
  \node       at (3+0.1, -4+0.1)  {\Cross} ;
  \node       at (3+0.1, -2+0.1)  {\Cross} ;
  \node[tick] at (1, -2)  {} ;
  \node[date] at (1, -2)  {0.} ;
  \node[date] at (1, -1)  {0.} ;
  \node[date] at (1, 0)  {0.} ;
  \node[date] at (1, -3)  {0.} ;
  \node[date] at (1, -4)  {0.} ;
  \node[date] at (2, 0)  {0.5} ;
  \node[tick] at (2, -4)  {} ;
  \node[date] at (2, -1)  {0.75} ;
  \node[date] at (2, -2)  {0.75} ;
  \node[date] at (2, -3)  {0.75} ;
  \node[date] at (2, -4)  {0.75} ;
  \node[tick] at (2, -3)  {} ;
  \node[date] at (3, -1)  {2.} ;
  \node[tick] at (3, -3)  {} ;
  \node[date] at (3, 0)  {1.33333333333} ;
  \node[date] at (3, -2)  {2.} ;
  \node[date] at (3, -3)  {2.} ;
  \node[date] at (3, -4)  {2.} ;
